pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -599,6 +599,10 @@ func (rc *reconciler) reconstructVolume(volume podVolume) (*reconstructedVolume,
 
 // updateDevicePath gets the node status to retrieve volume device path information.
 func (rc *reconciler) updateDevicePath(volumesNeedUpdate map[v1.UniqueVolumeName]*reconstructedVolume) {
+	if rc.kubeClient == nil {
```

Comment:
Can we create a unit test that will test the reconstruction flow?

Reply:
I ran through reconciler_test.go but it seems this func is not exercised by existing test.
Let me see how to run through this.

Reply:
Added new test where you would see the following without fix:
```
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xb0 pc=0x207971e]

goroutine 43 [running]:
testing.tRunner.func1(0xc0004b2300)
	/Users/yute/homebrew/Cellar/go/1.13.4/libexec/src/testing/testing.go:874 +0x3a3
panic(0x220bba0, 0x3265ab0)
	/Users/yute/homebrew/Cellar/go/1.13.4/libexec/src/runtime/panic.go:679 +0x1b2
k8s.io/kubernetes/pkg/kubelet/volumemanager/reconciler.(*reconciler).updateDevicePath(0xc00007eea0, 0x0)
	/Users/yute/go-workspace/src/k8s.io/kubernetes/pkg/kubelet/volumemanager/reconciler/reconciler.go:602 +0x3e
k8s.io/kubernetes/pkg/kubelet/volumemanager/reconciler.Test_UpdateStates(0xc0004b2300)
	/Users/yute/go-workspace/src/k8s.io/kubernetes/pkg/kubelet/volumemanager/reconciler/reconciler_test.go:94 +0x42b
testing.tRunner(0xc0004b2300, 0x24bca50)
```